packages/react-components/react-search/stories/SearchBox/SearchBoxBestPractices.md

```diff
@@ -2,4 +2,8 @@
 
 ### Do
 
+- **Consider using `SearchBox` with underline or outline appearances.** When the contrast ratio against the immediate surrounding color is less than 3:1, consider using underline or outline styles which has a bottom border stroke. But please ensure the color of bottom border stroke has a sufficient contrast which is greater than 3 to 1 against the immediate surrounding.
```

Comment:
This isn't something for this PR, but it'd be a nice quick change to add an item here for something like this:
> - **Consider wrapping SearchBox in a `role="search"` container for top-level site or app search fields.** This role exposes the search field as a landmark, and makes it easier to find for screen reader users. It should not be used unless the SearchBox is the primary search area for the entire site or app.